doc/build-unix.md

```diff
@@ -85,6 +85,7 @@ Now, you can either build from self-compiled [depends](/depends/README.md) or in
     sudo apt-get install libevent-dev libboost-system-dev libboost-filesystem-dev libboost-test-dev libboost-thread-dev
 
 BerkeleyDB is required for the wallet.
+To build, follow this [steps](https://github.com/bitcoin/bitcoin/blob/master/doc/build-openbsd.md#building-berkeleydb).
```

Comment:
The section in `doc/build-openbsd.md` you are linking to here is present in this file starting from line 184.

Reply:
Yes. k
My falut